filechanges/opt/musicbox/setsound.sh

```diff
@@ -179,33 +179,56 @@ then
 cat << EOF > /etc/asound.conf
 pcm.!default {
     type plug
+    slave.pcm plugequal;
+}
+pcm.plugequal {
+    type dmix
+    ipc_key 1024
+    controls "/home/mopidy/.alsaequal.bin"
```

Comment:
Here's some sample output of ```alsamixer -D equal```, but you'll have to run ```sudo apt-get install -y libasound2-plugin-equal``` first.

![mopidy ssh 180x46](https://cloud.githubusercontent.com/assets/10268911/8888708/e444550a-32b8-11e5-98c4-3c3dae26d8a3.png)